cf-agent/nfs.c

```diff
@@ -667,13 +667,23 @@ PromiseResult VerifyUnmount(EvalContext *ctx, char *name, Attributes a, const Pr
 static int MatchFSInFstab(char *match)
 {
     Item *ip;
+    const char delimit[]=" \t\r\n\v\f";
+    char *fstab_line, *token;
 
     for (ip = FSTABLIST; ip != NULL; ip = ip->next)
```

Comment:
I'd make the `FSTABLIST` a parameter of the function, so it's not single-purpose and so it doesn't rely on global variables.